GameOfLifeApp/Program.cs

```diff
@@ -1,2 +1,13 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using GameOfLifeApp;
+
+Console.Title = $"Game Of Life.";
+
+var field = new GameField(); 
+
+IApplicationManager trigger = new ApplicationManager(
+    new Communicator(),    
+    new GameLogic(),
+    new Serializer(),
+    new GameManager(new Communicator(), new Display(field), new GameLogic(), new Serializer()));
```

Comment:
you have 2 new communicators, that can be made into one.
you have 2 game logic that can be made one,
2 serializers.

Reply:
Perhaps I didn’t understand you but I used var for simplifying.